src/me/eglp/gv2/util/base/guild/config/GuildRemindersConfig.java

```diff
@@ -0,0 +1,120 @@
+package me.eglp.gv2.util.base.guild.config;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import me.eglp.gv2.main.Graphite;
+import me.eglp.gv2.util.base.guild.GraphiteGuild;
+import me.eglp.gv2.util.base.guild.reminder.GuildReminder;
+import me.eglp.gv2.util.base.guild.reminder.ReminderRepetitionEnum;
+import me.eglp.gv2.util.mysql.SQLTable;
+import me.mrletsplay.mrcore.misc.FriendlyException;
+
+/**
+ * This is the reminders Storage class, which also contains an interface to the
+ * DataBase for persistent reminder storage
+ * 
+ * @author The Arrayser
+ * @date Tue Mar 28 17:26:31 2023
+ */
+
+@SQLTable(name = "guilds_reminders", columns = { "GuildId varchar(255) NOT NULL", "`Id` varchar(255) NOT NULL",
+		"ChannelId varchar(255) NOT NULL", "Message text NOT NULL", "Repetition integer DEFAULT NULL",
+		"Date text NOT NULL", "LatestPossibleDate text NOT NULL",
+
+		"PRIMARY KEY (GuildId, `Id`)" }, guildReference = "GuildId")
+
+public class GuildRemindersConfig {
+
+	private GraphiteGuild guild;
+
+	List<GuildReminder> tempGuildReminders = new ArrayList<>();
```

Comment:
Just a minor nag, but I don't think `tempGuildReminders` is a good name. It implies that the items in the list are somehow temporary, even though they are just cached